### PR TITLE
TINY-8591: Add EditorManager page to documentation

### PR DESCRIPTION
**Pull request by TheSpyder**
Related Ticket: TINY-8591

Description of Changes:
* Uncommented the `@class` reference to generate a page for `EditorManager`
* Did this with a new page, instead of in `EditorManager.ts`, otherwise all of the references it adds are deleted from the `tinymce` root page.

Take a look at the branch on staging and see what we think:
http://docs-feature-62-tiny-8591.staging.tiny.cloud/docs/tinymce/6/apis/tinymce.editormanager/

The generated code isn’t great, this text is also used for the description so it ends up with `&#x60;` literals in the description. I've made a branch with the files checked in:
https://github.com/tinymce/tinymce-docs/compare/feature/6.2/TINY-8591

But I couldn’t figure out another way to do it with the way our moxiedoc/asciidoc code generation works.

Pre-checks:
* [x] ~Changelog entry added~
* [x] ~Tests have been added (if applicable)~
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] Docs ticket created (if applicable)

GitHub issues (if applicable):

### CODE REVIEW

**Review by MitchCTiny — APPROVED** [review]

**Review by ltrouton — APPROVED** [review]

File: modules/tinymce/tools/docs/tinymce.EditorManager.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/**
+ * This is the base class for the link:../tinymce.root/[`tinymce`] global, a factory for link:../tinymce.editor/[`tinymce.Editor`] instances.
+ *
+ * It is not generally used directly - the global link:../tinymce.root/[`tinymce`] is the only one expected to be created.
```
Comment (lnewson) [inline modules/tinymce/tools/docs/tinymce.EditorManager.js:4]: This doesn't really make sense, what's trying to be created? I'm not sure what it's trying to say though to be able to make a suggestion sorry.

**Review by lnewson — APPROVED** [review]
The output looks good, just not quite too sure on the wording of the part I commented on.

File: modules/tinymce/tools/docs/tinymce.EditorManager.js (hunk 1 above)
Comment (TheSpyder) [reply modules/tinymce/tools/docs/tinymce.EditorManager.js:4]: I'm trying to say we don't expect customers to create an instance of this class. We create one, which is the `tinymce` global, and that's all.

Comment (TheSpyder) [reply modules/tinymce/tools/docs/tinymce.EditorManager.js:4]: I've changed to "The global `tinymce` is the only one expected to be created." Is that better?

File: modules/tinymce/tools/docs/tinymce.EditorManager.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
+ * This is the base class for the link:../tinymce.root/[`tinymce`] global, a factory for link:../tinymce.editor/[`tinymce.Editor`] instances.
+ *
+ * The global link:../tinymce.root/[`tinymce`] is the only one expected to be created.
```
Comment (lnewson) [inline modules/tinymce/tools/docs/tinymce.EditorManager.js:4]:
```suggestion
 * The global link:../tinymce.root/[`tinymce`] is the only EditorManager instance expected to be created.
```
This clears it up I think, as `one` I thought was referring to editor instances and what caused part of my previous confusion.

Comment (lnewson) [reply modules/tinymce/tools/docs/tinymce.EditorManager.js:4]: @brian-forte might also have some other suggestions? We probably should have included him as a reviewer on this PR anyways given it's for docs.

Comment (brian-forte) [inline modules/tinymce/tools/docs/tinymce.EditorManager.js:4]: I’m reading this as if _link:../tinymce.root[`<string>`]_ are noun or adjectival phrases: denoters of discrete, identifiable things. Sort of, but not really, treating the strings as [bleeps](https://gocomics.com/peanuts/1964/11/04).

And, the editorial consequence of this reading is thus:

```suggestion
 * This is the base class for the link:../tinymce.root/[`tinymce`] global.
*
* This class is a factory for link:../tinymce.editor/[`tinymce.Editor`] instances.
*
* And the link:../tinymce.root/[`tinymce`] global is the only
* EditorManager instance the class is expected to create.
```

Comment (TheSpyder) [reply modules/tinymce/tools/docs/tinymce.EditorManager.js:4]: I'm not a fan of the bleeps and bloops but it's the only way to make moxiedoc + antora generate a link in the final rendering.

**Pull request merged**